formwidgets/blocks/assets/js/blocks.js

```diff
@@ -287,40 +308,53 @@
         })
     }
 
+    Blocks.prototype.onItemMouseOver = function(event) {
+        event.stopPropagation()
+
+        $(this.$el).find('.field-repeater-item').removeClass('hover')
+        $(event.currentTarget).closest('.field-repeater-item').addClass('hover')
+    }
+
+    Blocks.prototype.onItemMouseOut = function(event) {
+        event.stopPropagation()
+
+        $(event.currentTarget).closest('.field-repeater-item').removeClass('hover')
+    }
```

Comment:
Shouldn't this be done with focus instead of hover? i.e. if an element inside a block is focused that should be applying the style, not just applying the style because the mouse cursor is currently in a given block.

Reply:
I've applied both a hover and focus state. Focus will use the blue colour, hover uses a slightly darker grey.